lib/tasks/local_authorities.rake

```diff
@@ -0,0 +1,6 @@
+namespace :local_authorities do
```

Comment:
Similarly to the `import` module comment above, we could use an `import` namespace above `local_authorities` so that these are grouped with all the future ones.